compiler/optimizer/OMRRecognizedCallTransformer.cpp

```diff
@@ -55,6 +57,20 @@ int32_t OMR::RecognizedCallTransformer::perform()
    return 0;
    }
 
+bool OMR::RecognizedCallTransformer::isInlineable(TR::TreeTop* treetop)
+   {
```

Comment:
Who is the expected consumer of this API?  The optimizer inliner or a code generator?

Reply:
Its only consumer is `OMR::RecognizedCallTransformer::perform()`. The API is for downstream projects to provide information about whether a call node should be transformed. `OMR::RecognizedCallTransformer::perform()` applies transformation according to the return of this API.

Reply:
This is an ILGen Opt.